pkg/packages/manager.go

```diff
@@ -19,6 +20,7 @@ const (
 	retryLong      = time.Duration(60) * time.Second
 	retryVeryLong  = time.Duration(180) * time.Second
 	sourceRegistry = "sourceRegistry"
+	prefix         = api.PackageNamespace + "-"
```

Comment:
Is this needed?

Reply:
it gets used 3 places 🤷

Reply:
I mean why/where are namespaces being created with `-` in them?

Reply:
oh, that is just the namespace prefix. We will be creating these namespaces for each pbc. If there is a pbc for the billy cluster, there will be a namespace `eksa-packages-billy` for that cluster where packages CRs should go

Reply:
ooo ok, yea I think that's fine then.